stack/database.py

```diff
@@ -117,8 +144,8 @@
     DBInstanceClass=Ref(db_class),
     DBInstanceIdentifier=Ref(AWS_STACK_NAME),
     Engine="postgres",
-    EngineVersion="9.4.5",
-    MultiAZ=True,
+    EngineVersion=Ref(db_engine_version),
+    MultiAZ=Ref(db_multi_az),
```

Comment:
I think it's okay. the DBSubnetGroup must have at least two subnets, and you can't create an RDS instance in a VPC without a DBSubnet Group.

http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbsubnet-group.html
http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html